dydxV4/dydxV4/_Tracking/TrackingViewController+Ext.swift

```diff
@@ -48,7 +48,7 @@ extension TrackingViewController: ScreenIdentifiable, TrackingViewProtocol {
 private extension TrackingViewController {
     private var path: String {
         guard let path = history?.path else {
-            assertionFailure("no path for \(screenClass)")
+//            assertionFailure("no path for \(screenClass)")
```

Comment:
this is intentional. This line gets in the way rn when testing. It is intended to make sure we have 100% analytics path coverage so no worth the annoyance rn. Will come back to it